plat/layerscape/common/ls_common.mk

```diff
@@ -0,0 +1,78 @@
+#
+# Copyright (c) 2018, ARM Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+
+# For the original power-state parameter format, the State-ID can be encoded
+# according to the recommended encoding or zero. This flag determines which
+# State-ID encoding to be parsed.
+ARM_RECOM_STATE_ID_ENC := 0
```

Comment:
I don't think that you need to define `ARM_RECOM_STATE_ID_ENC` at all, that only affects Arm platforms code. Remove all the code related to it in this file.

Reply:
Removed, thanks.